microsoft/PowerToys
# [General]Default to English when a resource string is not found

## Summary of the Pull Request

When we've created new strings, it can take a while to get translations, meaning strings from resources.resx in C++ will get show weirdly:
![image](https://github.com/microsoft/PowerToys/assets/26118718/4b1eafab-f291-4ea8-9050-e08619151e34)

This PR uses an ATL helper to get the strings in en-us as a first fallback.


## Validation Steps Performed
Removed the `auto length = LoadStringW(instance, resource_id, reinterpret_cast<wchar_t*>(&text_ptr), 0);` code in a local build to make sure the first fallback was working.
Will now test on a CI build.
